pkg/client/client.go

```diff
@@ -698,6 +701,9 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 		if mode != "" {
 			fmt.Fprintf(tab, "  Mode:\t%s\n", mode)
 		}
+		if dsrMode != "" {
+			fmt.Fprintf(tab, "    DSR Mode:\t%s\n", dsrMode)
```

Comment:
```suggestion
			fmt.Fprintf(tab, "    DSR Dispatch Mode:\t%s\n", dsrMode)
```

Does this still format properly, or is the line becoming too wide?